core/src/main/java/org/bitcoinj/core/TransactionOutPointReference.java

```diff
@@ -72,28 +106,14 @@ public class TransactionOutPoint {
     public static TransactionOutPoint read(ByteBuffer payload) throws BufferUnderflowException, ProtocolException {
         Sha256Hash hash = Sha256Hash.read(payload);
         long index = ByteUtils.readUint32(payload);
-        return TransactionOutPoint.of(hash, index);
+        return TransactionOutPointReference.of(hash, index);
```

Comment:
Note that read is now returning the more compact `TransactionOutPoint` type (containing only hash and index)